lib/player.js

```diff
@@ -4778,7 +4809,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     let adCuePoints = [];
     if (this.adManager_) {
       try {
-        adCuePoints = this.adManager_.getServerSideCuePoints();
+        adCuePoints =
+            adCuePoints.concat(this.adManager_.getServerSideCuePoints());
+      } catch (error) {}
+      try {
+        adCuePoints =
+            adCuePoints.concat(this.adManager_.getMediaTailorCuePoints());
```

Comment:
Shouldn't this be abstracted by the ad manager? It seems out of place that the core player needs to know specifics about which ad provider is being used.

Reply:
Done!